src/Components/Footer/Footer.js

```diff
@@ -10,19 +10,19 @@ class Footer extends Component {
 		return (
 			// <!--Footer-->
 			<footer>
-				{/* Write your HTML here */}
+				{/* Write your HTML here */
 
 
 				<div className="row container-fluid text-center">
 
-					<ul className="col-md-9 row footer__links align-items-center" style={{ listStyleType: 'none' }}>
-						<li className="col-md my-2" ><a href="#">Contact Us</a></li>
-						<li className="col-md my-2"><a href="#">Udacity</a></li>
-						<li className="col-md my-2"><a href="#">Github Repos</a></li>
-						<li className="col-md my-2"><a href="#">Contributors</a></li>
-						<li className="col-md my-2"><a href="#">Send Feedback</a></li>
-						<li className="col-md my-2"><a href="#">Privacy Terms</a></li>
-					</ul>
+					<div className="col-md-9 row footer__links align-items-center">
```

Comment:
Any reason why these are `div` instead of `ul` and `li`?

Reply:
Yeah there's a reason. <li> tag takes the space for list-style-type which makes the footer irresponsive and two splitted lines instead of one **because of the list-style-type**. Though the type mentioned is _none_, it gives space to it making the error. Thus I replaced it with **div**.

Reply:
@goss-gowtham, have you tried `white-space: nwrap;` on that list? It might help with the problem that you are having.

Reply:
@abhishekakade , Yeah, I tried it. It takes an extra line to cover 5 links and Social ICONS.
[white-space:nowrap;](https://user-images.githubusercontent.com/29202347/39369906-1e1c9280-4a5b-11e8-9b7c-3bef8722506d.png) didn't seem to work. div tag is perfectly fine.

Reply:
@goss-gowtham Its fine, we'd want 2 lines at medium width so you can use something like this. There's not a whole lot difference in using `div` and `ul`, but it helps to structure out the code and that's why its preferable to use `li`s for "listing", semantic matters.
```
<ul className="col-md-9 row footer__links  align-items-center d-flex flex-row justify-content-lg-center">
    <li className="col-md my-2 align-self-center" ><a href="#">Contact Us</a></li>
    <li className="col-md my-2 align-self-center"><a href="#">Udacity</a></li>
    <li className="col-md my-2 align-self-center"><a href="#">Github Repos</a></li>
    <li className="col-md my-2 align-self-center"><a href="#">Contributors</a></li>
    <li className="col-md my-2 align-self-center"><a href="#">Send Feedback</a></li>
    <li className="col-md my-2 align-self-center"><a href="#">Privacy Terms</a></li>
</ul>
```
Also, there's something you need to change because there's a rogue style.
Please go to `Header.css` and change 
```
 li:first-of-type {
  margin-left: 0px;
}
```
to

```
#navbarNav li:first-of-type {
  margin-left: 0px;
}
```
To make it not interfere with the styling we have.
Make the changes accordingly and I'll have a look again

Reply:
So There's no problem if we have two lines in footer like 
[this](https://user-images.githubusercontent.com/29202347/39396701-d5edd08a-4b10-11e8-9401-020766e0d0b8.png)?

Reply:
For medium break-point, no.

Reply:
Are you talking about words in different line? If yes, then its fine for now. I was talking about width and resizing.